packages/mobile/screens/SignUp.tsx

```diff
@@ -0,0 +1,244 @@
+import { useMutation } from "@apollo/react-hooks";
+import { Field, Formik } from 'formik';
+import gql from "graphql-tag";
+import * as React from "react";
+import { Image, Text, TouchableOpacity, Alert} from 'react-native';
+import styled from 'styled-components/native';
+import * as Yup from "yup";
+import { TextInput } from "react-native-paper";
+import CREATE_USER_MUTATION from '../local_core/mutations/CREATE_USER_MUTATION';
+
+const signupSchema = Yup.object().shape({
+  username: Yup.string()
+    .required("Please enter your name")
+    .min(4, "Name is too short")
+    .max(12, "Thats a long name you have there"),
+  email: Yup.string()
+    .email("Please enter a valid email")
+    .required("Please enter an email"),
+  password: Yup.string().required("Please enter the password"),
+  confirmation: Yup.string()
+    .required("Please enter the password confirmation")
+    .oneOf([Yup.ref("password")], "Password does not match")
+});
+
+function alert(nombre : string)  {Alert.alert(
+  nombre,
+  "My Alert Msg",
+  [
+    {
+      text: "Cancel",
+      onPress: () => console.log("Cancel Pressed"),
+      style: "cancel"
+    },
+    { text: "OK", onPress: () => console.log("OK Pressed") }
+  ],
+  { cancelable: false }
+)};
+
+const SignUp = ({ navigation }: any) => {
+  const [signUpUser, { data }] = useMutation(CREATE_USER_MUTATION);
+
+  const submition = (values: any, { setErrors }: any) => {
+
+    signUpUser({
+      variables: {
+        username: values.username,
+        email: values.email,
+        password: values.password
+      }
+    })
+      .then(
+        ({
+          data: {
+            signedUser: { id }
+          }
+        }) => {
+          if (id && id > 0) {
+            navigation.navigate("HomeScreen");
+          }
+        }
+      )
+      .catch(({ Errors,graphQLErrors }) => {
+        const error = graphQLErrors?.map((err: any) => err?.message);
+          setErrors({ server: error[0] });
+      });
+  };
+
+
+
+  return (
+    <Container>
+      <Header>
+        <Image resizeMode={'cover'}
+          source={require('../assets/logo-lithium.png')}
+        />
+        <WelcomeText>{'Welcome to Lithium KB'}</WelcomeText>
+      </Header>
+      <Title>{'Sign up'}</Title>
```

Comment:
You can do this with <Title>Sign up</Title>

Reply:
ready